core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -216,6 +216,7 @@ public function create($blnHideUser=false)
 		{
 			$strDescription = $data['subject'];
 		}
+		$strDescription = StringUtil::substr($strDescription, System::getContainer()->get('database_connection')->getSchemaManager()->listTableColumns('tl_version')['description']->getLength());
```

Comment:
Why not read the length from the DCA? Would save us the DB query.

Reply:
:-/ any suggestion?

Reply:
@leofeyer How should I read the length from the DCA, without String parsing the `sql` field? Or ist that the idea, you had? We cannot rely on the `maxlength` value of the `eval` array. I have no clue how to get the length value directly than via the code @ausi proposed.

Reply:
@leofeyer Any proposal to get the conversation resolved?

Reply:
> String parsing the `sql` field? Or ist that the idea, you had?

Yes.

Reply:
We have a schema listener that does this and extracts the information to add it to the schema. It's okay to access it over the schema even if that means queries.